api_spec.yaml

```diff
@@ -2305,6 +2259,10 @@ paths:
                   $ref: '#/components/schemas/userDataInDatabase'
                 enableAntiCsrf:
                   $ref: '#/components/schemas/enableAntiCsrf'
+                useStaticSigningKey:
+                  type: boolean
+                  description: Decides if the token should be signed with a dynamic or static key, defaults to false
+                  example: false
       responses:
         '200':
           description: Create a new Session
```

Comment:
verifySession API also takes checkDatabase as a necessary boolean input? I think that's missing from this CDI spec